hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -91,7 +91,7 @@ public final class OzoneConfigKeys {
   public static final String DFS_CONTAINER_RATIS_DATASTREAM_ENABLE
       = "dfs.container.ratis.datastream.enable";
   public static final boolean DFS_CONTAINER_RATIS_DATASTREAM_ENABLE_DEFAULT
-      = true;
+      = false;
```

Comment:
Please check if the Streaming tests have set dfs.container.ratis.datastream.enable to true.  It seems TestBlockDataStreamOutput may not have it enabled?